lib/decors/refinements/unbound_method_visibility.rb

```diff
@@ -0,0 +1,15 @@
+module UnboundMethodVisibility
+    refine UnboundMethod do
```

Comment:
I don't get what is the advantage of using refinements, here. In this case it is much simpler to replace `decorated_method.visibility` by something similar to `Decors.method_visibibility(decorated_method)`.

I understand why you prefer to have `#visibility` defined on `UnboundMethod`, and I don't know  why it is not already defined in Ruby itself.  I also agree that `method_visibibility(decorated_method)` is less _sexy_, but it is way simpler to understand, it and doesn't need Ruby's refinements to work. Ruby is a sufficiently-complex language without that.

This article is interesting : http://interblah.net/why-is-nobody-using-refinements